### DISCUSSION TITLE
CodeQL Actions: C++ analysis runtime error

### DISCUSSION THREAD

**Discussion by asraa** [community/community#25346] [Actions]
<p>Hi there,</p>
<p>I’m integrating CodeQL into our project using GitHub Actions. I was able to get a successful custom build for our C++ project, but ran into an error during the analysis step in the CodeQL process:</p>
<pre><code class="lang-auto">2020-06-15T17:37:58.5293036Z [command]/opt/hostedtoolcache/CodeQL/1.0.0/x64/codeql/codeql database finalize /home/runner/work/_temp/codeql_databases/cpp
2020-06-15T17:37:59.7699264Z Running TRAP import for CodeQL database at /home/runner/work/_temp/codeql_databases/cpp...
2020-06-15T18:09:29.0621568Z Oops! A fatal internal error occurred.
2020-06-15T18:09:29.0630894Z java.lang.RuntimeException: java.lang.InternalError: a fault occurred in a recent unsafe memory access operation in compiled Java code
2020-06-15T18:09:29.0631049Z 	at com.semmle.util.exception.Exceptions.asUnchecked(Exceptions.java:103)
</code></pre>
<p>Here’s the logs and config for the run: <a href="https://github.com/asraa/envoy/runs/772810943?check_suite_focus=true" rel="nofollow noopener">https://github.com/asraa/envoy/runs/772810943?check_suite_focus=true</a></p>
<p>Does anyone have any pointers or can tell me where to start looking? Thanks!<br>
<a class="mention" href="/jhutchings1">@jhutchings1</a></p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/asraa">@asraa</a>,</p>
<p>Glad to see you in Github Forum ticket!<br>
It could be a JVM error which related to  <code>SIGBUG</code>  signal, it’s recommended to raise an issue in the <a href="https://github.com/github/codeql-action/issues" rel="nofollow noopener">github/codeql-action</a> to get further assistance, developing team help to check and answer.</p>
<p>Thanks.</p>
